caproto/_state.py

```diff
@@ -188,10 +200,13 @@ def _fire_state_triggered_transitions(self, role):
                                   self.circuit_state.states[role]))
         if new is not None:
             self.states[role], self.circuit_state.states[role] = new
+            return new
 
     def process_command(self, role, command_type):
-        self._fire_command_triggered_transitions(role, command_type)
-        self._fire_state_triggered_transitions(role)
+        command_trig = self._fire_command_triggered_transitions(role,
+                                                                command_type)
+        state_trig = self._fire_state_triggered_transitions(role)
+        return command_trig or state_trig
```

Comment:
What if a command triggers and transition (`command_trig` is not None) and then a state transition is triggered (`state_trig` is not None). Don't we want `state_trig` returned in that case?